bin/consequence_mapping/run_consequence_mapping.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# A wrapper script for running the consequence mapping pipeline concurrently and based on a VCF input.
+# See /docs/consequence-mapping.md for detailed information.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
If I had a dime for every time I have written this evil-but-necessary hack to get the working directory, I would be a millionaire...